causing/graph.py

```diff
@@ -138,11 +139,10 @@ def dot(
             else:
                 nodeff_str = ""
                 col_str = ""
-            if base_var:  # ToDo: if full_name # yyyy
-                xnode_show = xnode
-                # from apps.life import rr_report
-                # xnode_show = rr_report.full_name(xnode)
-                # xnode_show = rr_report.break_string(xnode_show)
+            if base_var:  # If full_name # yyy
+                xnode_show = (
+                    node_name[str(xnode)] if str(xnode) in node_name else str(xnode)
```

Comment:
Could also be written as
```suggestion
                    node_name.get(str(xnode), str(xnode))
```

Reply:
yes, thank you, had almost forgot about this :)